webui/react/src/pages/ExperimentDetails/ExperimentSingleTrialTabs.tsx

```diff
@@ -76,6 +84,7 @@ const ExperimentSingleTrialTabs: React.FC<Props> = ({
   const [canceler] = useState(new AbortController());
   const [trialDetails, setTrialDetails] = useState<TrialDetails>();
   const [tabKey, setTabKey] = useState(tab && TAB_KEYS.includes(tab) ? tab : DEFAULT_TAB_KEY);
+  const [remainingLogDays, setRemainingLogDays] = useState<Loadable<number>>(NotLoaded);
```

Comment:
sorry misunderstood the last comment. `remainingLogDays` type is number and undefined.